spec/Section 2 -- Language.md

```diff
@@ -716,28 +716,28 @@ sequences must be used to represent these characters.
 
 StringValue :: `""`
 
-  * Return an empty Unicode character sequence.
+  + Return an empty Unicode character sequence.
```

Comment:
Why change `*` to `+`? The `*` is fairly standard for bullet points and already renders correctly in GFM